.github/workflows/ci.yaml

```diff
@@ -12,28 +12,38 @@ jobs:
         id: info
         shell: bash
         run: |
+          echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
           echo "##[set-output name=node_version;]$(cat .nvmrc)"
+          echo "##[set-output name=yarn_cache;]$(yarn cache dir)"
```

Comment:
just an idea:
set the yarn cache to `$pwd/node_modules/global_cache` or sth using `yarn config set cache-folder <path>`
This way you'd only need one cache action and wouldn't need this variables.

Reply:
hmm... sounds like a good idea, although it will get very messy with jest cache, eslint cache, yarn cache, node_modules cache.

let hope that things get better when https://github.com/actions/cache/issues/16 gets resolved